src/utils/commitment_tree/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod binary_fact_tree;
 pub mod leaf_fact;
+pub mod nodes;
```

Comment:
i think we should remove the whole commitment tree thing for now. we'll have a good implementation soon